src/Main.java

```diff
@@ -1,10 +1,15 @@
-import client.DBConnection;
+import service.ConnectionService;
 import client.MainFrameController;
+import model.Session;
+import service.CustomerService;
 
 public class Main {
 
-
     public static void main(String[] args) {
-        MainFrameController mfc = new MainFrameController();
+        ConnectionService connectionService = new ConnectionService();
+        connectionService.connect("ora_j2z8", "a31484116");
```

Comment:
You seem to be hardcoding the username and password when I previously had it take it from the user input

Reply:
I think there's confusion about what is being 'logged into'. This is the connection to the database. The person using the app is logging into their account for our app - which is not the same as connecting to the database.

Reply:
True I guess so.